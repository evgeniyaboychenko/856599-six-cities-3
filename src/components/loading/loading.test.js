import React from 'react';
import renderer from 'react-test-renderer';
import Loading from './loading.jsx';

it(`should Loading render correctly`, () => {
  const tree = renderer.create(
      <Loading/>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
