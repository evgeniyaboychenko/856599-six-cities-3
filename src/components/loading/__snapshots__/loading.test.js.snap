// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should Loading render correctly 1`] = `
<div
  style={
    Object {
      "content": "",
      "marginLeft": "50%",
      "marginTop": "200px",
      "position": "absolute",
    }
  }
>
  <svg
    height="64px"
    version="1.0"
    viewBox="0 0 128 128"
    width="64px"
    xmlSpace="preserve"
    xmlns="http://www.w3.org/2000/svg"
    xmlnsXlink="http://www.w3.org/1999/xlink"
  >
    <g>
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="16"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="14.344"
        transform="rotate(45 64 64)"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="12.531"
        transform="rotate(90 64 64)"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="10.75"
        transform="rotate(135 64 64)"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="10.063"
        transform="rotate(180 64 64)"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="8.063"
        transform="rotate(225 64 64)"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="6.438"
        transform="rotate(270 64 64)"
      />
      <circle
        cx="16"
        cy="64"
        fill="#000000"
        fillOpacity="1"
        r="5.375"
        transform="rotate(315 64 64)"
      />
      <animateTransform
        attributeName="transform"
        calcMode="discrete"
        dur="720ms"
        repeatCount="indefinite"
        type="rotate"
        values="0 64 64;315 64 64;270 64 64;225 64 64;180 64 64;135 64 64;90 64 64;45 64 64"
      />
    </g>
  </svg>
</div>
`;
